pdaltools/add_points_in_pointcloud.py

```diff
@@ -6,6 +8,34 @@
 from pdaltools.las_info import get_tile_origin_using_header_info
 
 
+def parse_args(argv=None):
+    parser = argparse.ArgumentParser("TODO")
+    parser.add_argument("--input_geojson", "-ig", type=str, required=True, help="Input GeoJSON file")
+    parser.add_argument("--input_las", "-i", type=str, required=True, help="Input las file")
+    parser.add_argument("--output_las", "-o", type=str, required=False, default="", help="Output las file")
+    parser.add_argument(
+        "--virtual_points_classes",
+        "-c",
+        type=int,
+        default=66,
+        help="classification value to assign to those virtual points",
+    )
+    parser.add_argument(
+        "--spatial_ref",
+        type=str,
+        default="EPSG:2154",
+        help="spatial reference for the writer (required when running with a buffer)",
+    )
+    parser.add_argument(
+        "--tile_width",
+        type=int,
+        default=1000,
+        help="width of tiles in meters (required when running with a buffer)",
```

Comment:
```suggestion
        help="width of tiles in meters",
```
dans cette fonction il n'y a pas de buffer